lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -968,6 +980,14 @@ def _create_deployment_parameters(
         ]
         set_filtered_fields(self._azure_runbook, arm_parameters, copied_fields)
 
+        if self._azure_runbook.virtual_network_resource_group:
+            arm_parameters.virtual_network_resource_group = self._azure_runbook.virtual_network_resource_group
+        if self._azure_runbook.subnet_prefix:
+            arm_parameters.subnet_prefix = self._azure_runbook.subnet_prefix
```

Comment:
The subnet_prefix and virtual_network_name have default values in runbook, so it can be set always without checks.

Reply:
thanks!